### ISSUE TITLE
Config to disable hostname return from container

### ISSUE THREAD

**Issue by papapumpnz** [metalbear-co/mirrord#2395]
When stealing traffic from a container, any requests to obtain the hostname will return the hostname of the container being stole. 

Example (python 3.11):

```
from socket import gethostname

print (gethostname())
```

this will return the hostname of the container in the kubernetes cluster. It would be good to override this feature for times when you still want a call in your app to return the hostname of the workstation running the debug session, rather than the stolen container.

Example scenario:

Your application attempts to determine if it is running in a Kubernetes container or locally in your IDE to set some additional config depending on the outcome. If we always get the stolen containers hostname, we would have to introduce another way to determine if running locally in an IDE.

Other Options:

Dont use hostname as a variable to determine if running locally. Use something like checking for TTY, ie

```
import sys

if sys.stdout.isatty():
    print("Not running in an IDE")
else:
    print("Running in an IDE")
```

Not really ideal.

**Comment by aviramha** [comment]
Not sure if that'd be a suitable solution, but some people do
```json
{
{"feature": {"env": {"override": {"MIRRORD_IN_USE": "True"}}}
}
then check in code if mirrord is used with that (key can be anything ofc, also value)

**Comment by aviramha** [comment]
Implemented in #2396
It will be released on next minor version, probably Sunday.
